source/transfer_list.rst

```diff
@@ -902,5 +902,49 @@ This manifest contains the SPMC attribute node consumed by the SPMD
      - hdr_size
      - Holds a SPMC manifest image in DT format.
 
+**Read-Write Memory Layout Entry Layout (XFERLIST_RW_MEM_LAYOUT64)**
+
+This entry type holds a structure that describes the layout of a read-write
+memory region. TF-A defines this structure and uses it to convey to BL2 the
```

Comment:
Should the sentences after the first be written as an example? It seems that this information could be used elsewhere